src/routes/JobOpportunity/JobOpportunity.tsx

```diff
@@ -1,44 +1,17 @@
-import {
-  ActionButton,
-  FavoriteToggle,
-  LoginModal,
-  MainLayout,
-  RoutesNavigationList,
-  SimpleNavigationList,
-  Title,
-} from '@/components';
+import { components } from '@/api/schema';
 import { CompareCompetencesTable } from '@/components/CompareTable/CompareCompetencesTable';
-import { useEnvironment } from '@/hooks/useEnvironment';
+import OpportunityDetails, { type OpportunityDetailsSection } from '@/components/OpportunityDetails/OpportunityDetails';
 import { useToolStore } from '@/stores/useToolStore';
 import { getLocalizedText, getLocalizedTextByLang } from '@/utils';
-import { tidyClasses as tc } from '@jod/design-system';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
-import { MdCompareArrows, MdOutlinePrint, MdOutlineRoute, MdOutlineShare } from 'react-icons/md';
 import { useLoaderData } from 'react-router';
-import { LoaderData } from './loader';
-
-interface ScrollHeadingProps {
-  title: string;
-  heading: 'h1' | 'h2' | 'h3' | 'h4' | 'h5';
-  className: string;
-}
-const ScrollHeading = ({ title, heading, className }: ScrollHeadingProps) => {
-  const HeadingTag = heading as keyof JSX.IntrinsicElements;
-  return (
-    <HeadingTag id={title} className={tc(`scroll-mt-[96px] ${className}`)}>
-      {title}
-    </HeadingTag>
-  );
-};
+import { type LoaderData } from './loader';
```

Comment:
Is it necessary to explicitly tell the `type` here?

Reply:
I'm not sure if the compiler does it automatically, but I'm under the assumption that this is more efficient way of importing types/interfaces, as they get omitted from the final bundle.

Reply:
Yeah, it seems beneficial to explicitly tell the `type` after I read more about this.

It is also possible to use the following way, but it seems to have few gotchas so in general not recommended IIUC.
```tsx
import type { LoaderData } from './loader';
```



**So let's use like you already had done** 👍

Reply:
The mentioned approach won't work if a file exports mixed content, not just types. So you couldn't do `import type { FooType, FooComponent }`, but `import { type FooType, FooComponent }` works.

Reply:
Yeah, `import type` is only for types.

And the gotchas I mentioned, which in this particular case does not matter:

>Class Declarations: When importing a class using `import type`, you can’t use it to create instances or access static members

>Enums: Importing an enum with `import type` means you can't use its members as values